10-gis.Rmd

```diff
@@ -895,6 +897,70 @@ But the same is true for the lightweight SQLite/SpatiaLite database engine and G
 
 As a final note, if your data is getting too big for PostgreSQL/PostGIS and you require massive spatial data management and query performance, then the next logical step is to use large-scale geographic querying on distributed computing systems, as for example, provided by GeoMesa (http://www.geomesa.org/) or Apache Sedona [https://sedona.apache.org/; formermly known as GeoSpark - @huang_geospark_2017].
 
+
+
+
+
+## Bridges to cloud technologies and services {#cloud}
+
+
+
+### STAC, COGs, and data cubes in the cloud
+Major cloud computing providers (Amazon Web Services, Microsoft Azure / Planetary Computer, Google Cloud Platform, and others) offer huge catalogs of open Earth observation data such as the complete Sentinel-2 archive on their platforms. We can use R and directly connect to and process data from these archives, ideally from a machine in the same cloud and region.
+
+Three promising developments that make working with such image archives on cloud platforms _easier_ and _more efficient_ are the [SpatioTemporal Asset Catalog (STAC)](https://stacspec.org), the [cloud-optimized GeoTIFF (COG)](https://www.cogeo.org/) image file format, and the concept of data cubes. Below, we introduce these individual developments and briefly describe how they can be used from R. 
+
+The SpatioTemporal Asset Catalog (STAC) is a general description format for spatiotemporal data that is used to describe a variety of datasets on cloud platforms including imagery, SAR data, and point clouds. Besides simple static catalog descriptions, STAC-API presents a web service to query items (e.g. images) of catalogs by space, time, and other properties. In R, the **rstac** package [@simoes_rstac_2021] allows to connect to STAC-API endpoints and search for items. In the example below, we request all images from the [Sentinel-2 Cloud-Optimized GeoTIFF (COG) dataset on Amazon Web Services](https://registry.opendata.aws/sentinel-2-l2a-cogs) that intersect with a predefined area and time of interest. The result contains all found images and their metadata and URLs pointing to actual files on AWS. 
+
+```{r 09-stac-example, eval = FALSE}
+library(rstac)
+
+# Connect to the STAC-API endpoint for Sentinel-2 data
+# and search for images intersecting our AOI
+s = stac("https://earth-search.aws.element84.com/v0")
+items = s |>
```

Comment:
Looks good, note to self we need to switch to base pipe soon #616

Reply:
Base pipe FTW. (https://r4ds.hadley.nz/workflow-pipes.html)